app/models/project.rb

```diff
@@ -43,7 +43,7 @@ class Project < ActiveRecord::Base
   ## TODO: This potential has locking issues
   validates_with UniqueForOwnerValidator
 
-  before_save :send_notifications
+  before_save :send_notifications, if: Proc.new { |project| project.persisted? }
```

Comment:
yeah for sure. i didn't see after_update in the api docs list but it does exist.